web/app/sketcher/components/ConstraintEditor.jsx

```diff
@@ -30,7 +30,16 @@ export function ConstraintEditor() {
   const {viewer} = useContext(SketcherAppContext);
 
   const setValue = (name, value) => {
-    setValues({...value, [name]: value});
+    //setValues({...value, [name]: value});Bug! not working in case of object or multiple field like LockPoint. 
```

Comment:
```suggestion
    setValues({...values, [name]: value});
```

Reply:
@gauravkdeshmukh Looks great, I change it.